src/config/config.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"fmt"
+	"github.com/gin-gonic/gin"
+	env "github.com/caarlos0/env/v6"
+)
+
+// Config Available everywhere
+var Config *MainConfig
+
+type (
+	// MainConfig Type exported for use as input type in functions
+	MainConfig struct {
+		Port       string			`env:"PORT" envDefault:"8000"`
+		FileName   string			`env:"FILENAME" envDefault:"zap.log"`			
+		MaxSize    int				`env:"MAXSIZE" envDefault:"100"`
+		MaxAge     int				`env:"MAXAGE" envDefault:"10"`
+		MaxBackUp  int				`env:"MAZBACKUP" envDefault:"5"`
+		Compress   bool				`env:"COMPRESS" envDefault:"false"`
+		Level      string			`env:"LEVEL" envDefault:"info"`
+		OutputType string			`env:"OUTPUTTYPE" envDefault:"json"`
+	}
+)
+
+// LoadConfig Loads the config
+func LoadConfig(router *gin.Engine)  {
+	fmt.Println("Initializing LoadConfig")
+
+	cfg := MainConfig{}
+	if err := env.Parse(&cfg); err != nil {
+		panic(err)
+	}
+
+	// Address of parsed configs passing to Global Config variable
+	Config = &cfg
+
+	// Configuring Static Assets for development
+	router.Static("/assets", "../images")
```

Comment:
I believe we don't need favicon and assets for our APIs, will these be removed in future?

Reply:
Sure, The error when loading on the browser was getting as it was checking for the favicon. But got that when consuming as API we don't need this :-)